en/docs/learn/api-microgateway/deploy-apis-in-privatejet-mode.md

```diff
@@ -0,0 +1,321 @@
+# PrivateJet mode for Microgateways
+With many of applications gearing towards microservice architecture, it’s no surprise that container-orchestration systems such as Kubernetes have become so popular with functionalities such as automating computer application deployment, scaling, and management, 
+which simplifies a number of complex management tasks. WSO2 API-Manager provides cloud-native API management where user can expose microservices as managed APIs in cloud environments such as Kubernetes with the support of [WSO2-Kubernetes API Operator]({{base_path}}/learn/kubernetes-operators/k8s-api-operator.md). 
+
+ ![Architecture]({{base_path}}/assets/img/learn/privatejet-mode/architecture.png)
+ 
+Microservices will expose as managed APIs in cloud clusters in privetJet mode, where each microservice will have a dedicated [WSO2 API Microgateway](https://wso2.com/api-management/api-microgateway/). This will provide maximum security and guaranteed resource allocation for API execution. As depicted in the above diagram, When the APIs published via API-Manager in cloud environments,
+deployment, scaling, and management tasks will handle by the [WSO2 -Kubernetes API Operator]({{base_path}}/learn/kubernetes-operators/k8s-api-operator.md) itself.
+
+## Configuring APIs to deploy in PrivateJet mode
+
+Follow the steps below to configure the PrivateJet mode for Microgateways in API Manager to deploy APIs as managed APIs in cloud clusters. (Ex: Kubernetes)
+
+1. Install and setup [Kubernetes API Operator]({{base_path}}/learn/kubernetes-operators/k8s-api-operator.md). In order to expose APIs using ingress controller [deploy the k8s-api-operator in ingress mode](https://github.com/wso2/k8s-api-operator/tree/v1.1.0/scenarios/scenario-17). 
```

Comment:
We need to improve this part after we release the API Operator 1.2.0. We should mention all the steps for setting up this in K8s. Pointing scenario 17 is not enough. We will have to add scenario 17's prerequisites section here.

Reply:
updated with the changes